functions.php

```diff
@@ -53,11 +61,14 @@ function gesso_theme_scripts() {
 	// Maybe in 6.0.
 	$search_script_asset_file = include 'build/js/search.asset.php';
 	wp_enqueue_script( 'gesso-search', get_stylesheet_directory_uri() . '/build/js/search.js', $search_script_asset_file['dependencies'], $search_script_asset_file['version'] );
+	wp_enqueue_script( 'responsive-menu', get_stylesheet_directory_uri() . '/build/js/responsiveMenu.es6.js', array(), '1.0', true );
 }
 add_action( 'wp_enqueue_scripts', 'gesso_theme_scripts' );
 
 /**
  * Register block styles custom to Gesso theme.
+ *
+ * @return void
  */
 function gesso_block_assets() {
 	wp_enqueue_block_style(
```

Comment:
@tropicandid FYI all of these block styles are being registered without version numbers. This will cause problems with browser caching. https://developer.wordpress.org/reference/functions/wp_enqueue_block_style/

Reply:
Will address. Thank you for catching it!